src/storage/mvcc/txn.rs

```diff
@@ -223,7 +236,25 @@ impl<S: Snapshot> MvccTxn<S> {
             }
         }
 
+        let mut async_commit_ts = TimeStamp::zero();
+        if let Some(secondary_keys) = secondary_keys {
```

Comment:
When we are prewriting secondary keys (which means the `secondary_keys` argument is `None`), I think we still need to get `async_commit_ts`.

Reply:
> When we are prewriting secondary keys

In this case secondary keys will be Some([]), not None.

Reply:
> When we are prewriting secondary keys

In this case secondary keys will be Some([]), not None.